From 9426f36ed136e021b6c01913cb0e8bd95cd7a069 Mon Sep 17 00:00:00 2001
From: Tao Huang <huangtao@rock-chips.com>
Date: Fri, 16 Oct 2020 14:15:10 +0800
Subject: [PATCH] Revert "pwm: add rockchip-pwm.txt"

This reverts commit e623668278dc9e27a1892562bcf4cab7933ca2f3.
Remove unused Documentation/devicetree/bindings/pwm/rockchip-pwm.txt.

Change-Id: Ic938aea5aee8174f3c4d51bbe58a2a49c819ecc6
Signed-off-by: Tao Huang <huangtao@rock-chips.com>
---
 .../devicetree/bindings/pwm/rockchip-pwm.txt  | 31 -------------------
 1 file changed, 31 deletions(-)
 delete mode 100644 Documentation/devicetree/bindings/pwm/rockchip-pwm.txt

diff --git a/Documentation/devicetree/bindings/pwm/rockchip-pwm.txt b/Documentation/devicetree/bindings/pwm/rockchip-pwm.txt
deleted file mode 100644
index 74a0001e64bc..000000000000
--- a/Documentation/devicetree/bindings/pwm/rockchip-pwm.txt
+++ /dev/null
@@ -1,31 +0,0 @@
-Rockchip PWM controller
-
-Required properties:
-- compatible: should be "rockchip,<name>-pwm"
-- reg: physical base address and length of the controller's registers
-- #pwm-cells: should be 2.  The first cell specifies the per-chip index
-  of the PWM to use and the second cell is the period in nanoseconds.
-- pinctrl-names: must contain a "default" entry.
-- pinctrl-0: the pincontrol settings to configurethe.
-- clocks: phandle to the PWM source clock
-- clock-names: list of clock names sorted in the same order as the clocks.
-- status: for device state control (enabled/disabled).
-
-Example:
-
-pwm1: pwm@53fb4000 {
-	#pwm-cells = <2>;
-	compatible = "fsl,imx53-pwm", "fsl,imx27-pwm";
-	reg = <0x53fb4000 0x4000>;
-	interrupts = <61>;
-};
-pwm0: pwm@ff680000 {
-	compatible = "rockchip,rk-pwm";                                                                                                                                    
-	reg = <0xff680000 0x10>;                                                                                                                                           
-	#pwm-cells = <2>;                                                                                                                                                  
-	pinctrl-names = "default";                                                                                                                                         
-	pinctrl-0 = <&pwm0_pin>;                                                                                                                                           
-	clocks = <&clk_gates11 11>;                                                                                                                                        
-	clock-names = "pclk_pwm";                                                                                                                                          
-	status = "disabled";                                                                                                                                               
-};                                   
-- 
2.35.3

